builder/builder.js

```diff
@@ -37,41 +38,48 @@ module.exports = (options) => {
     cssDir,
     environment = 'development',
     jsDir,
-    output = defaultOutputThemeFile,
-    themeFile,
     watch,
   } = options;
 
-  const themeFiles = [defaultThemeYamlPath].concat(themeFile).filter(Boolean);
+  const bundles = [{
+    themeFiles: [defaultThemeYamlPath],
+    output: path.join(defaultOutputThemeDir, 'default'),
+  }];
+  getAvailableSets().forEach((themeOptions) => {
+    bundles.push({
+      themeFiles: [defaultThemeYamlPath].concat(getThemeFiles(themeOptions)),
+      output: path.join(defaultOutputThemeDir, `${themeOptions.brand}-${themeOptions.affiliate}`),
+    });
+  });
 
   const builder = themeBuilder({
     format: 'cssvars',
     prefix: 'tx',
   });
 
-  const outputThemeJs = path.join(
-    path.dirname(output),
-    `${path.basename(output, path.extname(output))}.js`
-  );
   const builderJS = themeBuilder({
     format: 'jsflat',
     prefix: 'tx',
   });
 
   if (watch) {
-    builder.watch(themeFiles, (result) => {
-      outputFile(output, result);
-    });
+    bundles.forEach(({ themeFiles, output }) => {
+      builder.watch(themeFiles, (result) => {
+        outputFile(`${output}.css`, result);
+      });
 
-    builderJS.watch(themeFiles, (result) => {
-      outputFile(outputThemeJs, `window.TravixTheme = ${JSON.stringify(result)};`);
+      builderJS.watch(themeFiles, (result) => {
+        outputFile(`${output}.js`, `window.TravixTheme = ${JSON.stringify(result)};`);
+      });
     });
   }
 
-  return Promise.all([
-    buildThemeCSS(builder, themeFiles, output),
-    buildThemeJS(builderJS, themeFiles, outputThemeJs),
-  ]).then(() => runWebpackAndCopyFilesToFinalDestination({
+  return Promise.all(bundles.reduce((result, { themeFiles, output }) => {
```

Comment:
And again here, could be simplified. It's fine for POC and we can read it of course, but... It takes time, a lot of nested stuff. Let's try to go more KISS way and write a bit simpler. Assume that your code will be maintained by babies 😄